targets/UnrealMarketplacePlugin/makeCpp.js

```diff
@@ -660,6 +661,21 @@ function getRequestActions(tabbing, apiCall, isInstanceApi) {
     return "";
 }
 
+function getCustomApiLogic(tabbing, api, apiCall, isInstanceApi) {
+    if (apiCall.name === "ExecuteFunction")
+        return tabbing + "FString localApiServer = PlayFabSettings::GetLocalApiServer();\n"
+            + tabbing + "TArray<FString> debugFunctions = PlayFabSettings::GetDebugFunctions();\n"
+            + tabbing + "if (!localApiServer.IsEmpty() && (debugFunctions.Contains(request.FunctionName) || debugFunctions.Contains(TEXT(\"*\"))))\n"
```

Comment:
Well if we do equals it would be more code cause then we'd have to make sure the size is 1 and that the first element is *, while to me it would make more sense to say * umbrellas everything else.

Reply:
What are you referring to in terms of function based settings?

Reply:
See my comment on https://github.com/nima-ap/SDKGenerator/pull/2